experiments/cifar10/train.py

```diff
@@ -45,6 +47,12 @@
     if FLAGS.n_folds > 0:
         X_train, Y_train = select_fold(X_train, Y_train)
 
+    # Make sure dims and current module name is included in the run log
+    # Note: this is currently kind of hackey, should clean up...
+    FLAGS.__flags['train_module'] = re.sub(r'\/', '.',
```

Comment:
Breaks if any experiment subdir contains "tanda" right?

Reply:
Also shouldnt this be somewhere in `train_scripts.py`?

Reply:
Fixed